packages/react-components/react-utilities-compat/library/README.md

```diff
@@ -1,5 +1,24 @@
 # @fluentui/react-utilities-compat
 
-**React Utilities components for [Fluent UI React](https://react.fluentui.dev/)**
+**React compatibility components for [Fluent UI React](https://react.fluentui.dev/)**
 
-These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+This package contains utility functions ported from Fluent UI React v8.
+This allows developers with high usage of v8 utility functions to migrate to v9.
+
+The utility functions are ported as-is with no improvements. Internal dependencies on v8 are ported into the internals folder and are only exported from the package if the utility function signature includes them.
+
```

Comment:
I'd remove the explicit recommendation on Lodash unless we strongly feel this is the right thing for most folks.

Reply:
Good point.